spec/integration/ktl_cluster_spec.rb

```diff
@@ -143,10 +182,10 @@
     context 'when given a topic regexp' do
       it 'only includes matched topics' do
         silence { run(%w[cluster preferred-replica ^topic1$], zk_args) }
-        expect(partitions).to match [
+        expect(final_state).to match [
```

Comment:
I don't understand what `match` does here. I'm looking at the [RSpec docs](https://relishapp.com/rspec/rspec-expectations/v/3-6/docs/built-in-matchers/match-matcher) and to me it looks like this should do `[a_hash_including(…)].match(final_state)`. AFAIK there is no `Array#match` so how does this work?

Maybe more importantly: how is this different from `expect(final_state).to include(a_hash_including(…))`?